docs/en/stack/ml/nlp/ml-nlp-shared.asciidoc

```diff
@@ -0,0 +1,41 @@
+tag::nlp-eland-clone-docker-build[]
+You can use the {eland-docs}[Eland client] to install the {nlp} model. Eland 
+commands can be ran via Docker. First, you need to clone the Eland repository 
+then create a Docker image of Eland:
+
+[source,shell]
+--------------------------------------------------
+git clone git@github.com:elastic/eland.git
+cd eland
+docker build -t elastic/eland .
+--------------------------------------------------
+
+After the script finished, your Eland Docker client is ready to use.
+end::nlp-eland-clone-docker-build[]
+
+tag::nlp-requirements[]
+To follow along the process on this page, you must have:
+
+* an {es} cluster that is set up properly to use the {ml-features}. Refer to 
+<<setup>>.
```

Comment:
Since the eland commands assume you're using a Cloud environment, that might be worth mentioning here too.